### ISSUE TITLE
Roads

### ISSUE THREAD

**Issue by Sneakydeaky** [Bouncyrock/TaleSpire-Beta-AssetRequests#6]
**Describe the Asset**
the assets i mainly want are just better roads at the moment. like a 1 tile that the basic road (so double wide roads could be made and with that half tile roads. like half road half grass. some stone roads would be excellent as well something that as worn down stone and has wagon tracks  to signal a busy city with lots of trade. or overgrown roads to show that this place has seen little travel.

**Comment by ScottinNH** [comment]
Diagonal and curved road tiles.

**Comment by ShaunDave** [comment]
Agreed... would be nice to use a blank dirt road tile as the base of the 'town square' in a small town that wouldn't spend money on cobblestones
